SeleniumWebDriver/SeleniumWebDriver/TutByLoginTest/Pages/LoginPage.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Threading;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumWebDriver.TutByLoginTest.Pages
+{
+    public class LoginPage
+    {
+        private const string URL = "https://tut.by/";
+        private const string _loginPopUpButton = "enter";
+        private const string _loginName = "login";
+        private const string _passwordName = "password";
+        private const string _loginButton = ".m-green";
+        private IWebDriver _driver;
+
+        IWebElement _LoginPopUpButton;
+        IWebElement _Login;
+        IWebElement _Password;
+        IWebElement _Enter;
+
+        //move driver to LoginPage, naviagate to URL and Initialize WebElements
+        public LoginPage(IWebDriver driver)
+        {
+            _driver = driver;
+
+            //implictwaiter
+            _driver.Manage().Timeouts().ImplicitWait = System.TimeSpan.FromSeconds(5);
+
+            _driver.Navigate().GoToUrl(URL);
+            Initialize();
+        }
+
+        //initialize WebElements
+        public void Initialize()
+        {
+            _LoginPopUpButton = _driver.FindElement(By.ClassName(_loginPopUpButton));
+            _Login = _driver.FindElement(By.Name(_loginName));
+            _Password = _driver.FindElement(By.Name(_passwordName));
+            _Enter = _driver.FindElement(By.CssSelector(_loginButton));
+        }
```

Comment:
This is not effective way to work with web elements on all page. I would recommend you to call each element separately in according methods. It is important because you cannot be 100% sure that part of your web elements will not be changed after some time because of any actions on page of application. So just call necessary elements in time.

Reply:
fixed with "fd87310"